WIP-CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+Changed:
```

Comment:
We list the requirements in the README as well. Not sure if this is the best of ideas, but for now update it there as well.